src/components/tech-skills.tsx

```
@@ -0,0 +1,88 @@
+  SiTailwindcss,
+  SiFigma,
+  SiGit 
+} from "react-icons/si"
+
+export default function TechnologiesSection() {
+  return (
+    <section className="py-16 ">
+      <div className="container px-4 mx-auto">
+        <h2 className="text-3xl font-bold mb-2">Current technologies</h2>
+        <p className="text-gray-600 mb-10">
+          I'm proficient in a range of modern technologies that empower me to build highly functional solutions. These are some of my <b>Main</b> technologies.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)